site/docs/components/accordion/usage.mdx

```diff
@@ -34,4 +34,18 @@ import {
 
 ### Props
 
+#### Accordion
+
 <PropsTable componentName="Accordion" />
+
+#### Accordion Group
+
+<PropsTable componentName="AccordionGroup" />
```

Comment:
```suggestion
<PropsTable packageName="core" componentName="AccordionGroup" />
```

Reply:
Don't think this is required as core is default